### ISSUE TITLE
Make left side (file list) scrollable.

### ISSUE THREAD

**Issue by mnott** [PDF-Archiver/PDF-Archiver#3]
Hi,

The filenames easily get long. It should be possible to scroll the list horizontally, so as to see an entire file name.

**Comment by JulianKahnert** [comment]
Hey,
I see your point - a quick workaround: resize the document name column. You can scroll afterwards to see the complete filename.

**Comment by sandersantema** [comment]
I've got little to none knowledge of swift and very little of programming in general but I've tried to get this to work just for fun. I didn't succeed (such a surprise 😉) but I would like to know how this could be done. As far as I know the left table is stored in a NSScrollview which should be configurable to allow scrolling, both vertically and horizontally, however after searching for a while and changing everything which could be relatable I still haven't got it to work. I'd like to know how you did it if you succeed. It might be better to allow resizing instead of scrolling but that seemed a lot more complicated so I decided not to try and work out how that would work.
